client/galaxy/scripts/mvc/workflow/workflow-view-terminals.js

```diff
@@ -221,7 +228,119 @@ var BaseOutputTerminalView = TerminalView.extend({
+                            .addClass("delete-terminal")
+                            .attr("tabindex", "0")
+                            .attr("aria-label", "delete terminal")
+                            .on("keydown click", e => {
+                                if (e.keyCode === 32 || e.type === "click") {
+                                    //Space or Click
+                                    $.each(inputTerminal.connectors, (_, x) => {
+                                        if (x) {
+                                            x.destroy();
+                                            ariaAlert("Connection destroyed");
+                                        }
+                                    });
+                                    t.remove();
+                                }
+                            });
+                        $(currentItem.input)
+                            .parent()
+                            .append(t);
+                    }
+                    break;
+            }
+        };
+        const buildInputChoicesMenu = () => {
+            const inputChoicesMenu = document.createElement("ul");
+            $(inputChoicesMenu).focusout(e => {
+                /* focus is still inside child element of menu so don't hide */
+                if (inputChoicesMenu.contains(e.relatedTarget)) {
+                    return;
+                }
+                $(inputChoicesMenu).hide();
+            });
+            inputChoicesMenu.id = "input-choices-menu";
+            inputChoicesMenu.className = "list-group";
+            inputChoicesMenu.setAttribute("role", "menu");
+            this.$el.attr("aria-grabbed", "true");
+            this.$el.attr("aria-owns", "input-choices-menu");
+
+            $(".input-terminal").each((i, el) => {
+                const input = $(el);
+                const inputTerminal = input.context.terminal;
+                const connectionAcceptable = inputTerminal.canAccept(this.el.terminal);
+                if (connectionAcceptable.canAccept) {
+                    const inputChoiceItem = document.createElement("li");
+                    inputChoiceItem.textContent = `${inputTerminal.name} in ${inputTerminal.node.name} node`;
+                    inputChoiceItem.tabIndex = -1;
+                    inputChoiceItem.input = input;
+                    inputChoiceItem.onkeydown = inputChoiceKeyDown;
+                    inputChoiceItem.className = "list-group-item";
+                    inputChoiceItem.setAttribute("role", "menuitem");
+                    inputChoicesMenu.appendChild(inputChoiceItem);
+                }
+            });
+            if (inputChoicesMenu.firstChild) {
+                this.$el.append(inputChoicesMenu);
+                inputChoicesMenu.firstChild.classList.add("active");
+                inputChoicesMenu.firstChild.focus();
+            } else {
+                //alert user that there are no available inputs to connect to for this output
```

Comment:
@WilliamHolden Need to add something here?

Reply:
Thanks, I added the alert text